src/renderer/components/ft-community-post/ft-community-post.vue

```diff
@@ -115,11 +115,39 @@
     <div
       class="bottomSection"
     >
-      <span class="likeCount"><font-awesome-icon
-        class="thumbs-up-icon"
-        :icon="['fas', 'thumbs-up']"
-      /> {{ voteCount }}</span>
-      <span class="commentCount">
+      <span
+        class="likeCount"
+        :title="$tc('Global.Counts.Like Count', voteCount, {count: formattedVoteCount})"
+        :aria-label="$tc('Global.Counts.Like Count', voteCount, {count: formattedVoteCount})"
+      >
+        <font-awesome-icon
+          class="thumbs-up-icon"
+          :icon="['fas', 'thumbs-up']"
+          aria-hidden="true"
+        /> {{ formattedVoteCount }}</span>
+      <router-link
+        v-if="isInvidiousAllowed && !singlePost"
+        :to="`/post/${postId}`"
```

Comment:
If the `authorId` is available it makes sense to pass it along, so that we don't need to make the extra API call to resolve the post URL to get the channel ID.
```suggestion
        :to="{
          path: `/post/${postId}`,
          query: authorId ? { authorId } : undefined
        }"
```